profile_report.py

```diff
@@ -0,0 +1,31 @@
+from profiler import data
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("benchmark", type=str)
+parser.add_argument("path", type=str)
+
+args = parser.parse_args()
+
+# Processes the set of previously written logs
+
+threshold = 1.10  # Percent difference
+
+db = data.FileBasedProfileDB(args.path)
+actual_max_ts = 0
+dt = db.find(f"{args.benchmark}")
+c = 0
+
+if len(dt) >= 2:
+    exec_time = [0.0, 1.0]
+    last_two = dt[-2:]
+    exec_time[0] = last_two[0].user_time_sec
+    exec_time[1] = last_two[1].user_time_sec
+
+    if threshold * float(exec_time[1]) < float(exec_time[0]) or float(exec_time[1]) > threshold * float(exec_time[0]):
+        raise SystemExit(f"Potential performance degradation detected {exec_time[0]} vs {exec_time[1]}")
+    print(f"Prev vs Curr execution time {exec_time[0]} vs {exec_time[1]}")
+    print("No recent performance degradation detected")
+    print(
+        f"TileDB version prev = {dt[0].tiledbsoma_version} curr = {dt[1].tiledbsoma_version}"
+    )
```

Comment:
might want to output this before the check, since it's relevant to the user in the case of an alert